api/src/main/java/com/cloud/event/EventTypes.java

```diff
@@ -1172,4 +1172,8 @@ public static Class getEntityClassForEvent(String eventName) {
 
         return null;
     }
+
+    public static boolean isVpcEvent(String eventType) {
+        return EventTypes.EVENT_VPC_CREATE.equals(eventType) || EventTypes.EVENT_VPC_DELETE.equals(eventType);
```

Comment:
not for update?

Reply:
@DaanHoogland from my point of view, as the VPC update API only changes metadata such as name, MTU, etc., I see no need to handle the update event here.